app/src/standard/java/com/infomaniak/mail/firebase/ProcessMessageNotificationsWorker.kt

```diff
@@ -63,21 +65,22 @@ class ProcessMessageNotificationsWorker @AssistedInject constructor(
             SentryDebug.sendFailedNotification("No messageUid in Notification", SentryLevel.ERROR, userId, mailboxId)
             return@withContext Result.success()
         }
+
+        // Refresh User
+        AccountUtils.updateCurrentUser()
+
+        // Refresh Mailboxes
+        SentryLog.d(TAG, "Refresh mailboxes from remote")
+        with(ApiRepository.getMailboxes()) {
+            if (isSuccess()) mailboxController.updateMailboxes(data!!)
```

Comment:
If the call fails, we will have the same comportment as before, do we want to send a sentry if it failed for another reason than Network ?